src/components/Shared/SendModal/SendOnChain/SendOnChain.test.tsx

```diff
@@ -73,20 +71,20 @@ describe("SendOnChain", () => {
   test.skip("validates the address-input for BTC address format", async () => {
     setup();
 
-    const addressInput = screen.getByLabelText(
-      "wallet.address"
-    ) as HTMLInputElement;
+    let addressInput = screen.getByLabelText("wallet.address");
 
     userEvent.clear(addressInput);
     userEvent.type(addressInput, "abc123456789");
 
-    await waitFor(() => expect(addressInput).toHaveClass("input-error"));
+    addressInput = await screen.findByLabelText("wallet.address");
+
+    // expect(addressInput).toHaveClass("input-error");
```

Comment:
Should this be removed?

Reply:
Whoops was not intended, was Just Testing It out